src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/ChallengeParticipantsWidget.java

```diff
@@ -0,0 +1,93 @@
+package org.sagebionetworks.web.client.widget.entity.renderer;
+
+import java.util.Map;
+
+import org.sagebionetworks.repo.model.UserProfile;
+import org.sagebionetworks.web.client.ChallengeClientAsync;
+import org.sagebionetworks.web.client.SynapseClientAsync;
+import org.sagebionetworks.web.client.utils.Callback;
+import org.sagebionetworks.web.client.widget.WidgetRendererPresenter;
+import org.sagebionetworks.web.client.widget.pagination.DetailedPaginationWidget;
+import org.sagebionetworks.web.client.widget.pagination.PageChangeListener;
+import org.sagebionetworks.web.shared.UserProfilePagedResults;
+import org.sagebionetworks.web.shared.WidgetConstants;
+import org.sagebionetworks.web.shared.WikiPageKey;
+
+import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+public class ChallengeParticipantsWidget implements ChallengeParticipantsView.Presenter, WidgetRendererPresenter, PageChangeListener {
+	
+	private ChallengeParticipantsView view;
+	private Map<String,String> descriptor;
+	private ChallengeClientAsync challengeClient;
+	private String challengeId;
+	private boolean isInTeam;
+	private Callback widgetRefreshRequired;
+	private DetailedPaginationWidget paginationWidget;
+	public static final Long DEFAULT_PARTICIPANT_LIMIT = 50L;
+	public static final Long DEFAULT_OFFSET = 0L;
+	
+	@Inject
+	public ChallengeParticipantsWidget(ChallengeParticipantsView view, 
+			DetailedPaginationWidget paginationWidget, 
+			ChallengeClientAsync synapseClient) {
+		this.view = view;
+		this.paginationWidget = paginationWidget;
+		this.challengeClient = synapseClient;
+		view.setPaginationWidget(paginationWidget.asWidget());
+		view.setPresenter(this);
+	}
+	
+	@Override
+	public void configure(final WikiPageKey wikiKey, final Map<String, String> widgetDescriptor, Callback widgetRefreshRequired, Long wikiVersionInView) {
+		this.descriptor = widgetDescriptor;
+		this.widgetRefreshRequired = widgetRefreshRequired;
+		challengeId = descriptor.get(WidgetConstants.CHALLENGE_ID_KEY);
+		isInTeam = false;
```

Comment:
default to null.  verify that null returns all participants.  if set in markdown, then abide.